frontend/src/components/ConnectWalletButton.tsx

```diff
@@ -35,7 +35,7 @@ export const ConnectWalletButton = ({
 
   useEffect(() => {
     const walletName = getWalletName();
-    console.log(walletName)
+    console.log(walletName);
```

Comment:
When pushing to a repository, it's important to remove any consoles before pushing or making a pull request.